master/reference/calicoctl/resources/ippool.md

```diff
@@ -53,4 +53,12 @@ subnet.  The subnet of each node is configured on the node resource (which may b
 determined when running the calico/node service).
 
 For details on configuring IP-in-IP on your deployment, please read the
-[Configuring IP-in-IP guide]({{site.baseurl}}/{{page.version}}/usage/configuration/ip-in-ip).
\ No newline at end of file
+[Configuring IP-in-IP guide]({{site.baseurl}}/{{page.version}}/usage/configuration/ip-in-ip).
+
+> **NOTE**
+>
+> Routing between Workloads and Hosts running Calico is asymmetric when
+IPIP is enabled, this causes
+[RPF](https://en.wikipedia.org/wiki/Reverse_path_forwarding) checks to fail
+and the traffic to be discarded between them. The current work around is to
+enable nat-outgoing or to use the tunnel address on the host.
```

Comment:
"To use the tunnel address on the host" isn't very clear.

I don't think we need to go into all the detail here - why don't we just say that that it's asymmetric and that in most cases you will always want IPIP and NAT-outgoing enabled.  If you enable IPIP without NAT outgoing, traffic may be filtered on the reverse-path.